### PR TITLE
Update to nom4

### PR DESCRIPTION
**Pull request by kpcyrd**
This superseeds #19 with the patch I posted as a comment.

### CODE REVIEW

**Comment by badboy** [comment]
Could you rebase on top of master? I pushed a change to .travis.yml, so that this PR will actually run.

**Comment by kpcyrd** [comment]
Done!

**Comment by badboy** [comment]
Thanks! Tests passed. I'll give this a look later today and then hopefully merge it.

File: tests/lib.rs [hunk 1]
```diff
@@ -3,57 +3,62 @@
 extern crate iso8601;
```
Comment (badboy) [inline tests/lib.rs]: Only realizing now that all the parsers stuff is still public.
We should probably move the tests here into becoming unit tests inside the crate and replace it here with proper integration tests of the (smaller) public API. 

I'll do that in a follow-up though.

**Comment by kpcyrd** [comment]
Awesome! Any chance you could publish this as a new version? The debian freeze is rapidly approaching and this would allow us to ship exa in debian buster without applying a 85kb patch to iso8601. :)

Thanks!

**Comment by badboy** [comment]
On it! Either tonight or tomorrow during the day I'll push it.

**Comment by badboy** [comment]
done: https://crates.io/crates/iso8601

**Comment by sylvestre** [comment]
And in Debian: https://tracker.debian.org/news/1026863/accepted-rust-iso8601-030-1-source-into-unstable/

**Comment by badboy** [comment]
Nice!

**Pull request merged**